packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -99,8 +100,8 @@ contract SystemConfig is OwnableUpgradeable, ISemver {
     uint256 public startBlock;
 
     /// @notice Semantic version.
-    /// @custom:semver 1.7.1
-    string public constant version = "1.7.1";
+    /// @custom:semver 1.7.2
+    string public constant version = "1.7.2";
```

Comment:
Both semver changes should be minor and not patch because the changes impact the bytecode and not just the source code